code/datums/spells/bloodcrawl.dm

```diff
@@ -131,20 +132,27 @@
 		playsound(get_turf(L), sound, 100, 1)
 		sleep(3 SECONDS)
 
+	var/mindless = (ishuman(victim) || isrobot(victim))
```

Comment:
If you're only making this check once, I'd say you don't need this variable and can instead just check the condition where you currently check the variable

Reply:
Yeah, that is a relic of when the code is less optimised. I'll change that in a bit.